Updates to CAVE install page:

srcarter3 opened the pull request:
- add new subsection for each OS, about how to upgrade CAVE if already installed
- add new subsection at the end for each OS about how to completely remove CAVE
- add new subsection for windows system-wide installer for the access form

Update Common Problems page to use "macOS" section title, to be consistent with our Install CAVE page

Commit:
Updates to CAVE install page:
- add new subsection for each OS, about how to upgrade CAVE if already installed
- add new subsection at the end for each OS about how to completely remove CAVE
- add new subsection for windows system-wide installer for the access form

Update Common Problems page to use "macOS" section title, to be consistent with our Install CAVE page

tiffanycmeyer13 commented on docs/install/install-cave.md:
```diff
@@ -51,6 +51,12 @@ Regardless of what Operating System CAVE is running on, these general requiremen
 
 !!! note "While CentOS8 has reach End of Life as of Dec. 31, 2021, CentOS7 End of Life isn't until June 30, 2024."
 
+### Upgrade Existing Installation
```
I feel like this is confusing...

"Whether you already have CAVE installed or not, you can follow the [Download and Installation Instructions](#download-and-installation-instructions) below. If installed the script will remove the old version and install the latest."

srcarter3 replied:
Okay yup, that's better.

tiffanycmeyer13 commented on docs/install/install-cave.md:
```diff
@@ -51,6 +51,12 @@ Regardless of what Operating System CAVE is running on, these general requiremen
+If you do not currently have CAVE installed, please go directly to the [Download and Installation Instructions](#download-and-installation-instructions).
+
+If you already have CAVE installed, following the [Download and Installation Instructions](#download-and-installation-instructions) below will also properly take care of your existing install.
+
```
If you just want to remove the existing version of CAVE and not re-install, follow the [uninstall instructions](#linux).

srcarter3 replied:
I did actually have that in there...but then was wondering if we should do it for every OS then (have that sentence linking to the proper section at the bottom), and then I opted just to remove it instead of including it in all the sections?

tiffanycmeyer13 replied:
Yeah...Linux is different than Windows and Mac so I think the link to fully remove is needed for sure here

srcarter3 replied:
Ok sounds good 👍

tiffanycmeyer13 commented on docs/install/install-cave.md:
```diff
@@ -208,3 +255,17 @@ sudo yum remove awips2-*
+
+To completely remove CAVE:
+
+1. Type "remove" in the search bar and select the first option that comes up.  This will open the Applications settings.
```
I would say:
Type "remove" in the search bar and select "Add or remove programs" because if the user has a local file named remove that could come up first...and I confirmed on my Windows 10 VM and Windows 11 it was the same

Commit:
Few small updates to install cave page based on Tiffany's feedback:
- simplified text for linux install with existing cave
- added link to remove cave for linux install
- clarified search bar instructions for windows steps to remove cave
